pkg/scheduler/listers/listers.go

```diff
@@ -33,8 +34,16 @@ type PodLister interface {
 	FilteredList(podFilter PodFilter, selector labels.Selector) ([]*v1.Pod, error)
 }
 
-// NodeLister interface represents anything that can list/get node object from node name.
-type NodeLister interface {
-	// TODO(ahg-g): rename to Get and add a List interface.
-	GetNodeInfo(nodeName string) (*v1.Node, error)
+// NodeInfoLister interface represents anything that can list/get NodeInfo objects from node name.
+type NodeInfoLister interface {
+	// Returns the list of NodeInfos.
+	List() ([]*schedulernodeinfo.NodeInfo, error)
+	// Returns the NodeInfo of the given node name.
+	Get(nodeName string) (*schedulernodeinfo.NodeInfo, error)
+}
+
+// SharedLister groups scheduler-specific listers.
+type SharedLister interface {
+	Pods() PodLister
```

Comment:
How about renaming function name to `PodLister()` and `NodeInfoLister`? The meaning seems more clear when calling these functions.